deploy/manifests/base/assigner/deployment.yaml

```
@@ -8,11 +8,14 @@ spec:
   selector:
     matchLabels:
       app: assigner
+  strategy:
```

Minor comment: Document rationale? something like:
```suggestion
  # Terminate previous assigner before rolling out new ones to avoid conflicts between assignments.
  strategy:
```